docs/cli.md

```diff
@@ -870,6 +876,26 @@ grid stop run [OPTIONS] RUN_NAMES...
 | ---- | ---- | ----------- | ------- |
 | `--help` | boolean | Show this message and exit. | `False` |
 
+### grid stop session
+
+Stop one or more SESSION_NAMES.
+
+This preserves progress completed up to this point, but stops
+further computations and any billing for the machines used.
+
+**Usage:**
+
+```
+grid stop session [OPTIONS] SESSION_NAMES...
+```
+
+**Options:**
+
+| Name | Type | Description | Default |
+| ---- | ---- | ----------- | ------- |
+| `--help` | boolean | Show this message and exit. | `False` |
+
+
```

Comment:
@pritamsoni-hsr these are all really good changes that will be over-written, so we need to make these changes in the template that generate the CLI docs. 
cc @oojo12